ogr/services/github.py

```diff
@@ -721,6 +721,21 @@ def create_release(self, tag: str, name: str, message: str) -> Release:
         )
         return self.get_release(created_release.id)
 
+    def edit_release(self, identifier: int, name: str, message: str):
```

Comment:
I am not sure if the identifier is the right input. (Also in the `get_release` method since it is some internal number, and we do not have it even in the [Release class](https://github.com/packit-service/ogr/blob/master/ogr/abstract.py#L213).)
I am also not sure if this is something we have in the other git forges (Pagure, Gitlab).

I know it would not be as easy as with id, but we need to get releases by tag (ideally) or by name.

What about the following methods:

- `get_releases() -> Release`
- `get_last_release() -> List[Release]`
- `get_release(identifier=None, name=None, tag=None) -> Release` (you need to set one and only one, leave the identifier for backward compatibility)
- I do not know what would be a best way to update release. What about adding it directly to `Release` class:

- [ ] Make a `GithubRelease` as a subclass of `Release` which will contain release representation of PyGithub.
- [ ] Create `GithubRelease` instances instead of `Release` in `GithubProject`.
- [ ] Add the `edit_release` as an abstract method to the `Release` class and implement it in the `GithubRelease`.

Anyway, we have a plan in #86 to implement all Issue/PR/Release related methods directly in those objects, so why not start with this now.